main/src/library/FileWrite.flix

```diff
@@ -100,9 +107,9 @@ mod FileWrite {
     ///
     /// Handles the `FileWrite` effect of the given function `f`.
     ///
-    /// In other words, re-interprets the `FileWrite` effect using the `IO` effect.
+    /// In other words, re-interprets the `FileWrite` effect using the `Sys` and `IO` effects.
     ///
-    pub def handle(f: a -> b \ ef): a -> Result[IoError, b] \ (ef - FileWrite) + IO = x ->
+    pub def handle(f: a -> b \ ef): a -> Result[IoError, b] \ (ef - FileWrite) + {Sys, IO} = x ->
```

Comment:
I would like to avoid the Sys effect here. So lets just call `getTemporaryDirectory` directly from Java.